addons/l10n_in/tests/common.py

```diff
@@ -0,0 +1,131 @@
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+class L10nINTestInvoicingCommon(AccountTestInvoicingCommon):
+    @staticmethod
+    def setup_common_india(clss):
```

Comment:
```suggestion
    def setup_common_india(cls):
```

Reply:
I named it 'clss' because runbot fails due to ruff (https://docs.astral.sh/ruff/rules/bad-staticmethod-argument/)

Reply:
Alright, I learned something, thanks ! 
I'd still rename it though (IDK, indian_test_setup?) so it doesn't look like a typing error :)

Reply:
afterwards it will LGTM @jbw-odoo